app/src/main/java/com/asfoundation/wallet/home/ui/TestHomeFragment.kt

```diff
@@ -0,0 +1,90 @@
+package com.asfoundation.wallet.home.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.unit.dp
+import com.appcoins.wallet.core.utils.android_common.Log
+import com.appcoins.wallet.ui.arch.SingleStateFragment
+import com.appcoins.wallet.ui.widgets.TopBar
+import com.asf.wallet.R
+import com.asfoundation.wallet.viewmodel.BasePageViewFragment
+
+//TODO to be removed. for testing
```

Comment:
is it going to be merged or should remove before?

Reply:
the idea here was to keep the test fragment until the new Home fragment is made. The test fragment will then be replaced by the new HomeFragment (in the feature branch). Same for the below comments